grpc/send_user_friend_request.go

```diff
@@ -24,9 +24,9 @@ func (s *gRPCFriendsServer) SendUserFriendRequest(ctx context.Context, in *pb.Se
 
 	message := in.GetMessage()
 
-	id := database_wiiu.SaveFriendRequest(sender, recipient, sentTime.Value(), expireTime.Value(), message)
+	id, _ := database_wiiu.SaveFriendRequest(sender, recipient, sentTime.Value(), expireTime.Value(), message)
```

Comment:
This should probably have a more robust error check, similar to `AcceptFriendRequest`